threatstream_connector.py

```diff
@@ -312,8 +312,21 @@ def _process_response(self, r, action_result):
 
         return RetVal(action_result.set_status(phantom.APP_ERROR, message), None)
 
+    def _update_header(self, headers=None):
```

Comment:
Is the default needed? It's a new function so there is no need for it due to compatibility reasons.

Reply:
Some of the actions don't pass headers in `make_rest_call` function. After the new changes in API, authorization headers will be required. Instead of changing it all places, we have handled it in the updated_headers function. For the same we have added the default value.